xinference/deploy/cmdline.py

```diff
@@ -259,12 +259,14 @@ def list_model_registrations(
 @click.option("--size-in-billions", "-s", default=None, type=int)
 @click.option("--model-format", "-f", default=None, type=str)
 @click.option("--quantization", "-q", default=None, type=str)
+@click.option("--peft-model-path", "-p", default=None, type=str)
```

Comment:
`-p` means path in most cases, which may not be a proper short option name for peft model ID.